app/services/description_generator.rb

```diff
@@ -108,50 +109,6 @@ def contact
     )
   end
 
-  sig { returns(T::Array[Cocina::Models::Contributor]) }
-  def contributors
-    work.contributors.map do |work_form_contributor|
-      contributor(work_form_contributor)
-    end
-  end
-
-  # in cocina model terms, returns a DescriptiveValue
-  sig { params(work_form_contributor: Contributor).returns(Cocina::Models::Contributor) }
-  def contributor(work_form_contributor)
-    # TODO: we may know status primary
-    if work_form_contributor.person?
-      Cocina::Models::Contributor.new(
-        name: [
-          {
-            value: "#{work_form_contributor.last_name}, #{work_form_contributor.first_name}"
-          }
-        ],
-        type: work_form_contributor.contributor_type,
-        # TODO: we will know code, uri, source code and source uri
-        role: [
-          {
-            value: work_form_contributor.role
-          }
-        ]
-      )
-    else
-      Cocina::Models::Contributor.new(
-        name: [
-          {
-            value: work_form_contributor.full_name
-          }
-        ],
-        type: work_form_contributor.contributor_type,
-        # TODO: we will know code, uri, source code and source uri
-        role: [
-          {
-            value: work_form_contributor.role
-          }
-        ]
-      )
-    end
-  end
-
```

Comment:
all this moved to the new contributors_generator class

Reply:
love it